src/js/components/form-elements/v2/Switch.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+
+import PropTypes from 'prop-types';
+
+import Translate from 'utils/Translate';
+
+import './style.scss';
+
+const Switch = ({ defaultValue, titles, onChange }) => {
+  const [value, changeValue] = useState(defaultValue);
+
+  const onChangeValue = () => {
+    changeValue((previousValue) => {
+      onChange?.(!previousValue);
+      return !previousValue;
+    });
+  };
+
+  return (
+    <div id="switch-container">
```

Comment:
Why you are using `id` here? What if there are going to be multiple switch fields on the form for some usecase?